google-cloud-spanner/src/main/java/com/google/cloud/spanner/GrpcStreamIterator.java

```diff
@@ -57,15 +60,20 @@ class GrpcStreamIterator extends AbstractIterator<PartialResultSet>
   GrpcStreamIterator(
       Statement statement, int prefetchChunks, boolean cancelQueryWhenClientIsClosed) {
     this.statement = statement;
+    this.prefetchChunks = prefetchChunks;
     this.consumer = new ConsumerImpl(cancelQueryWhenClientIsClosed);
     // One extra to allow for END_OF_STREAM message.
-    this.stream = new LinkedBlockingQueue<>(prefetchChunks + 1);
+    this.stream = new LinkedBlockingQueue<>((prefetchChunks * 2) + 1);
```

Comment:
I don't think we should increase the buffer size for this change (although now that I see this, I also better understand why the above implementation of `onStreamMessage` works)

Reply:
Do we still need to increase the buffer size?